renderdoc/driver/gl/gl_hooks_egl.cpp

```diff
@@ -509,6 +505,12 @@ __attribute__((visibility("default"))) EGLBoolean eglSwapBuffers(EGLDisplay dpy,
   eglhooks.eglQuerySurface_real(dpy, surface, EGL_HEIGHT, &height);
   eglhooks.eglQuerySurface_real(dpy, surface, EGL_WIDTH, &width);
 
+  GLInitParams &init = eglhooks.GetDriver()->GetInitParams();
+  int colorspace;
+  eglhooks.eglQuerySurface_real(dpy, surface, EGL_COLORSPACE, &colorspace);
+  // GL_SRGB8_ALPHA8 is specified as color-renderable, unlike GL_SRGB8.
+  init.isSRGB = init.colorBits == 32 && colorspace == EGL_COLORSPACE_sRGB;
```

Comment:
I think the constants are different (from the egl.h ver 1.5):
```
#define EGL_COLORSPACE                    0x3087
#define EGL_COLORSPACE_sRGB               0x3089
```
```
#define EGL_VG_COLORSPACE                 0x3087
#define EGL_VG_COLORSPACE_sRGB            0x3089
```
```
#define EGL_GL_COLORSPACE                 0x309D
#define EGL_GL_COLORSPACE_SRGB            0x3089
```
from the EGL_KHR_gl_colorspace extension:
```
EGL_GL_COLORSPACE_KHR                   0x309D
EGL_GL_COLORSPACE_SRGB_KHR              0x3089
```

That aside: having the colorpsace as a defult 0 is good imho.